substdio.c

```diff
@@ -1,11 +1,6 @@
 #include "substdio.h"
 
-void substdio_fdbuf(s,op,fd,buf,len)
-register substdio *s;
-register int (*op)();
-register int fd;
-register char *buf;
-register int len;
+void substdio_fdbuf(substdio *s, ssize_t (*op)(int,void*,size_t), int fd, char *buf, int len)
```

Comment:
This doesn’t give warnings if `op`’s second arg happens to be `const void *`?

Reply:
No, because it is (wrongly) just in the C file. I have removed the argument specification in the header, so it has to go away here, too.

Reply:
In fact, I will leave it as is. See below.